recipe/meta.yaml

```diff
@@ -22,11 +21,42 @@ requirements:
     - ninja
   host:
     - openssl
+    - libopenssl-static
 
-test:
-  commands:
-    - test -f $PREFIX/lib/libs2n${SHLIB_EXT}  # [unix]
-    - test -f $PREFIX/include/s2n.h  # [unix]
+outputs:
+  - name: s2n
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - cmake
+        - {{ compiler('c') }}
+        - ninja
+    test:
+      commands:
+        - test -f $PREFIX/lib/libs2n${SHLIB_EXT}  # [unix]
+        - test ! -f $PREFIX/lib/libs2n.a  # [unix]
+        - test -f $PREFIX/include/s2n.h  # [unix]
+        - if not exist %LIBRARY_BIN%\libs2n.dll (exit 1)  # [win]
+        - if not exist %LIBRARY_LIB%\libs2n.lib (exit 1)  # [win]
+  - name: s2n-static
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - cmake
+        - {{ compiler('c') }}
+        - ninja
+      host:
+        - openssl
+        - libopenssl-static
```

Comment:
pin `s2n` here exactly.

Reply:
Now linux fails, but I'm not sure it's the right way to pin `s2n`.